core-bundle/src/Resources/contao/library/Contao/Email.php

```diff
@@ -120,13 +124,21 @@ class Email
 	/**
 	 * Instantiate the object and load the mailer framework
 	 *
-	 * @param \Swift_Mailer|null $objMailer
+	 * @param \Swift_Mailer|MailerInterface|null $objMailer
 	 */
-	public function __construct(\Swift_Mailer $objMailer = null)
+	public function __construct($objMailer = null)
 	{
-		$this->objMailer = $objMailer ?: System::getContainer()->get('swiftmailer.mailer');
+		$this->objMailer = $objMailer ?: System::getContainer()->get('mailer');
```

Comment:
That implies that the Symfony mailer is able to handle a `\Swift_Message()` object. Intended?

Reply:
Depending on the mailer instance, different message object will be created below.